evalml/model_understanding/visualizations.py

```diff
@@ -419,6 +446,53 @@ def graph_prediction_vs_actual_over_time(pipeline, X, y, X_train, y_train, dates
         dates,
     )
 
+    if is_multiseries(pipeline.problem_type):
```

Comment:
nit: can we wrap the original non-multiseries case in the else block and only return at the end of the method? Helpful for code clarity.